release_dashboard/test/widgets/engine_cherrypicks_substeps_test.dart

```diff
@@ -2,20 +2,49 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'package:conductor_core/proto.dart' as pb;
+import 'package:conductor_ui/state/status_state.dart';
+import 'package:conductor_ui/widgets/common/url_button.dart';
 import 'package:conductor_ui/widgets/engine_cherrypicks_substeps.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_test/flutter_test.dart';
+import 'package:provider/provider.dart';
+
+import '../src/services/fake_conductor.dart';
 
 void main() {
+  const String conductorVersion = 'v1.0';
```

Comment:
We're duplicating existing code. We should create something like `test/src/entity_generators.dart` and have a helper function like `generateConductorState` with all these values, but we can make them all optional.